.travis.yml

```diff
@@ -1,3 +1,9 @@
+# Travis CI is a Continuous Integration environment that helps detecting problems
+# and test failures at each push/pull-request automatically.
```

Comment:
Given that people will be using this as a learning tool I would be explicit when possible. 
> push and pull-request